R/run_prep_calculation_loans.R

```diff
@@ -318,6 +328,9 @@ run_prep_calculation_loans <- function(input_path_project_specific,
 
   # ADO 1933 - for now, this only includes sectors with production pathways
   # in the future, sectors with emissions factors based pathways may follow
+  # tmsr <- scenario_data_market_share %>%
```

Comment:
no commented out code please (refers to all)

Reply:
removed